### PR TITLE
Add onramp extra properties to analytics events

### PR DESCRIPTION
**Pull request by wachunei**
This PRs adds more properties to some Fiat On Ramp Aggregator analytics events

### CODE REVIEW

File: app/components/UI/FiatOnRampAggregator/Views/Checkout.tsx [hunk 1]
```diff
@@ -155,6 +155,12 @@ const CheckoutWebView = () => {
         trackEvent('ONRAMP_PURCHASE_SUBMITTED', {
```
Comment (alaahd) [inline app/components/UI/FiatOnRampAggregator/Views/Checkout.tsx:155]: Non-blocking: for better readability, can you move the tracking code into an external function.

**Review by alaahd — APPROVED** [review]

File: app/components/UI/FiatOnRampAggregator/Views/Checkout.tsx (hunk 1 above)
Comment (wachunei) [reply app/components/UI/FiatOnRampAggregator/Views/Checkout.tsx:155]: It is better to have full visibility of the payload of each event, this is helpful for data team when QAing

**Review by gantunesr — APPROVED** [review]
LTGM

**Comment by cortisiko** [comment]
Spoke with Lorenzo and he confirmed this was QA passed. See discussion here https://consensys.slack.com/archives/C039U5608K0/p1657634239853159?thread_ts=1657131054.888789&cid=C039U5608K0

**Pull request merged**